client/my-sites/plugins/plugin-remove-button/index.jsx

```diff
@@ -57,9 +58,76 @@ module.exports = React.createClass( {
 		}
 	},
 
-	renderButton: function() {
-		var inProgress = PluginsLog.isInProgressAction( this.props.site.ID, this.props.plugin.slug, [ 'REMOVE_PLUGIN' ] );
+	getDisabledInfo() {
+		if ( ! this.props.site ) { // we don't have enough info
+			return null;
+		}
+
+		if ( ! this.props.site.hasMinimumJetpackVersion ) {
```

Comment:
Since all this checks & error messages are shared with the toggles, we should extract them to a helper (something like we do with access-control.js